build.sbt

```diff
@@ -7,7 +7,7 @@ lazy val commonSettings = Seq(
   organizationName := "Evolution",
   organizationHomepage := Some(url("https://evolution.com")),
   scalaVersion := crossScalaVersions.value.head,
-  crossScalaVersions := Seq("2.13.5", "2.12.13"),
+  crossScalaVersions := Seq("2.13.14", "3.3.3"),
```

Comment:
```suggestion
  crossScalaVersions := Seq("2.13.15", "3.3.3"),
```

Reply:
it could be useful to add one more file to this PR - copy of https://github.com/evolution-gaming/safe-akka/blob/master/.scala-steward.conf - to prohibit Scala 3 update